acc_Bias: [-0.1763622, -0.1469346, -0.1260973]

acc_Ea: [         1, -0.001202348, 0.001565606, 
                  0,          1, 0.006864399, 
                  0,          0,          1]

acc_Ka: [  1.002226,   0.998947,   1.005947]

gyro_Bias: [0.002865561, 0.006743124, 0.003355583]

gyro_Eg: [         1, -0.00281022, -0.0005060587, 
          0.00701874,          1, 0.01224008, 
          -5.324506e-05, -0.005338015,       1]

gyro_Kg: [  1.002103,  0.9932116,  0.9945976]
